definitions/variable/emissions/emissions.yaml

```diff
@@ -1,27 +1,55 @@
+# 1996 categories: https://github.com/primap-community/climate_categories/blob/main/climate_categories/data/IPCC1996.yaml
 
 - Emissions|{Level-1 Species}:
-    description: Emissions of {Level-1 Species}
+    description: Emissions of {Level-1 Species} to the atmosphere. 
+      For the avoidance of doubt, these emissions are net emissions 
+      (i.e. include both positive (emission) and negative (removal) 
+      contributions. For positive contributions only, see 
+      "Gross Emissions|*").
     unit: "{Level-1 Species}"
     components:
       - Emissions|{Level-1 Species}|AFOLU
       - Emissions|{Level-1 Species}|Energy
       - Emissions|{Level-1 Species}|Industrial Processes
       - Emissions|{Level-1 Species}|Waste
-      - Emissions|{Level-1 Species}|Capture and Removal
+      - Emissions|{Level-1 Species}|Other Capture and Removal
+      - Emissions|{Level-1 Species}|Product Use
       - Emissions|{Level-1 Species}|Other
 - Gross Emissions|CO2:
-    description: Gross emissions of carbon dioxide (CO2), not accounting for negative
-      emissions from bioenergy with CCS (BECCS) or agriculture, forestry
-      and other land use (AFOLU)
+    description: Gross emissions of carbon dioxide (CO2). 
+      This is emissions sources only. All removals (for example, 
+      removals from BECCS or direct air carbon capture and storage (DACCS)) 
+      are not included here. For net emissions, see "Emissions|*".
     unit: Mt CO2/yr
 
 - Emissions|{Level-2 Species}|AFOLU:
     description: Emissions of {Level-2 Species} from agriculture, forestry
-      and other land use (IPCC category 3)
+      and other land use (IPCC 1996 category 4, 5; IPCC 2006 category 3), 
+      reporting the sum of sources and sinks including land-use methods that do not 
+      use CCS such as soil carbon sequestration, biochar, forestry, peatland 
+      and wetland restoration, and biomass burial. Emissions reported in this 
+      category represent fluxes to/from the atmosphere from/to the land pool.
     unit: "{Level-2 Species}"
+- Gross Emissions|CO2|AFOLU:
+  description: Gross emissions of carbon dioxide (CO2)  from agriculture, forestry
+      and other land use (IPCC 1996 category 4, 5; IPCC 2006 category 3).
+      This is emissions sources only. All removals (for example, 
+      removals from biochar) are not included here. For net emissions, 
+      see "Emissions|*". Emissions reported in this category represent 
+      fluxes from the land pool to the atmosphere.
+  unit: Mt CO2/yr
+- Gross Emissions|CO2|AFOLU|Deforestation:
+  description: Gross emissions of carbon dioxide (CO2) from deforestation
+- Gross Emissions|CO2|AFOLU|Other Land:
+  description: Gross emissions of carbon dioxide (CO2) from other land-use change
+- Gross Emissions|CO2|AFOLU|Forest Management:
+  description: Gross emissions of carbon dioxide (CO2) from forest management
```

Comment:
There are no corresponding net-emissions variables for deforestation, other-land and forest-management. I suggest to remove these variables for now and then have a follow-up PR and coordinate with the AFOLU team.

Reply:
OK now in https://github.com/IAMconsortium/common-definitions/pull/199